src/test/java/org/openmrs/contrib/qaframework/automation/ConditionsSteps.java

```diff
@@ -1,17 +1,19 @@
 package org.openmrs.contrib.qaframework.automation;
 
-import io.cucumber.java.After;
-import io.cucumber.java.Before;
-import io.cucumber.java.en.And;
-import io.cucumber.java.en.Given;
-import io.cucumber.java.en.Then;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
 import org.apache.commons.lang3.StringUtils;
 import org.openmrs.contrib.qaframework.RunTest;
 import org.openmrs.reference.page.ConditionPage;
```

Comment:
Change the import statemement package to import from https://github.com/openmrs/openmrs-contrib-qaframework/tree/master/src/test/java/org/openmrs/contrib/qaframework/page to cater for the current changes

Reply:
alright thanks


> to cater for the current changes

Reply:
hey @sherrif10 , the changes by [this Pull request](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/582) are not captured here-->  https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/src/test/java/org/openmrs/contrib/qaframework/page/ConditionsPage.java